src/output/icons.rs

```diff
@@ -544,15 +560,15 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "out"            => '\u{eb2c}',              // 
     "p12"            => Icons::KEY,              // 
     "par"            => Icons::COMPRESSED,       // 
-    "part"           => '\u{f43a}',              // 
+    "part"           => Icons::DOWNLOAD,         // 󰇚
     "patch"          => Icons::DIFF,             // 
     "pbm"            => Icons::IMAGE,            // 
     "pdf"            => '\u{f1c1}',              // 
     "pem"            => Icons::KEY,              // 
     "pfx"            => Icons::KEY,              // 
     "pgm"            => Icons::IMAGE,            // 
-    "phar"           => Icons::LANG_PHP,         // 
-    "php"            => Icons::LANG_PHP,         // 
+    "phar"           => Icons::LANG_PHP,         // 
+    "php"            => Icons::LANG_PHP,         // 
```

Comment:
I'm also not sure about this one

Reply:
as someone who regularly must look at these file extensions... I approve of the elephant